Makefile

```diff
@@ -1,3 +1,6 @@
+.PHONY: build
```

Comment:
I'm confused, shouldn't phony be specific to the target (`all`) instead of the filename (`build`)?

Reply:
Oops, good catch! I meant to name it `build` instead of `all`, silly typo.

Reply:
This was my saddest review ever, I was honestly expecting to learn some new makefile trick :(